model/LoggerService.php

```diff
@@ -65,7 +67,7 @@ public static function logEvent(Event $event)
             $event->getName(),
             $context->getRequest()->getRequestURI(),
             $currentUser->getIdentifier(),
-            join(',', $currentUser->getRoles()),
+            join(',', $currentUser->getPropertyValues(PROPERTY_USER_ROLES)),
             (new DateTime())->format(DateTime::ISO8601),
```

Comment:
Yes, it is. But not here. It is a date for the database, but in view we should use date helper.